code/modules/mob/living/living_say.dm

```diff
@@ -228,6 +228,15 @@ GLOBAL_LIST_INIT(message_modes_stat_limits, list(
 		message = "[randomnote] [message] [randomnote]"
 		spans |= SPAN_SINGING
 
+	// Leaving this here so that anything that handles speech this way will be able to have spans affecting it and all that.
+	var/sigreturn = SEND_SIGNAL(src, COMSIG_MOB_SAY, args, message_range)
```

Comment:
In #69844, I initially tried to do `SEND_SIGNAL(src, COMSIG_MOB_SAY, args, message_range, saymode)` but it kept saying the list index was out of bounds when I tried to access `SPEECH_SAYMODE`  I don't believe `SEND_SIGNAL` was merging the latter arguments correctly in the marco:

`#define SEND_SIGNAL(target, sigtype, arguments...) ( !target.comp_lookup?[sigtype] ? NONE : target._SendSignal(sigtype, list(target, ##arguments)) )`

When I was testing `SPEECH_RANGE` before my changes, it wouldn't showup in the args list.  Not to mention that `SPEECH_FILTERPROOF` was skipped accidentally.

The only place that touches SPEECH_RANGE is dullhans, so this might fix say code involving them.